contracts/flat/BentoBoxFlat.sol

```diff
@@ -0,0 +1,1064 @@
+// SPDX-License-Identifier: UNLICENSED
+// The BentoBox
```

Comment:
Why is the BentoBox included here?
Should it not only be imported through the package?